### PR TITLE
a bunch of java docs

### PR DESCRIPTION
**Pull request by piercetrey-figure**
Here's a bunch of java docs, I tried to be consistent and clear, but your opinion may vary 😄

### CODE REVIEW

File: os-client/src/main/kotlin/io/provenance/scope/objectstore/client/CachedOsClient.kt [hunk 1]
```diff
@@ -76,6 +94,13 @@ class CachedOsClient(val osClient: OsClient, osDecryptionWorkerThreads: Short, o
         )
     }
 
+    /**
+     * Fetch a jar from Object Store by hash
+     * @param [hash] the hash of the jar
+     * @param [encryptionKeyRef] the [PrivateKey][java.security.PrivateKey] to use to decrypt the jar
```
Comment (scirner22) [inline os-client/src/main/kotlin/io/provenance/scope/objectstore/client/CachedOsClient.kt:100]: There's a few places where the `PrivateKey` is mentioned alongside the `encryptionKeyRef` param, should we instead note that the key ref is rather used as a encryption/decryption provider?

**Comment by scirner22** [comment]
:awesome:

**Review by scirner22 — APPROVED** [review]

File: os-client/src/main/kotlin/io/provenance/scope/objectstore/client/CachedOsClient.kt (hunk 1 above)
Comment (piercetrey-figure) [reply os-client/src/main/kotlin/io/provenance/scope/objectstore/client/CachedOsClient.kt:100]: oh yeah, good catch, I'll go through those for consistency

Comment (piercetrey-figure) [reply os-client/src/main/kotlin/io/provenance/scope/objectstore/client/CachedOsClient.kt]: pushed that up

**Pull request merged**